com.microsoft.mrtk.core/Editor/Inspectors/StatefulInteractableInspector.cs

```diff
@@ -10,57 +10,57 @@ namespace Microsoft.MixedReality.Toolkit.Editor
     {
-        private SerializedProperty isToggled;
-        private SerializedProperty isToggledStateActive;
-        private SerializedProperty selectThreshold;
-        private SerializedProperty deselectThreshold;
-        private SerializedProperty toggleMode;
-        private SerializedProperty triggerOnRelease;
+        private SerializedProperty IsToggled;
+        private SerializedProperty IsToggledStateActive;
+        private SerializedProperty SelectThreshold;
+        private SerializedProperty DeselectThreshold;
+        private SerializedProperty ToggleMode;
+        private SerializedProperty TriggerOnRelease;
         private SerializedProperty allowSelectByVoice;
         private SerializedProperty speechRecognitionKeyword;
-        private SerializedProperty voiceRequiresFocus;
-        private SerializedProperty useGazeDwell;
-        private SerializedProperty gazeDwellTime;
-        private SerializedProperty useFarDwell;
-        private SerializedProperty farDwellTime;
-        private SerializedProperty onClicked;
-        private SerializedProperty onToggled;
-        private SerializedProperty onDetoggled;
-        private SerializedProperty onEnabled;
-        private SerializedProperty onDisabled;
+        private SerializedProperty VoiceRequiresFocus;
+        private SerializedProperty UseGazeDwell;
+        private SerializedProperty GazeDwellTime;
+        private SerializedProperty UseFarDwell;
+        private SerializedProperty FarDwellTime;
+        private SerializedProperty OnClicked;
+        private SerializedProperty OnToggled;
+        private SerializedProperty OnDetoggled;
+        private SerializedProperty OnEnabled;
+        private SerializedProperty OnDisabled;
         private static bool advancedFoldout = false;
         private static bool enabledEventsFoldout = false;
 
         protected override void OnEnable()
         {
             base.OnEnable();
 
-            isToggled = SetUpProperty(nameof(isToggled));
-            isToggledStateActive = isToggled.FindPropertyRelative("active");
+            IsToggled = SetUpAutoProp(nameof(IsToggled));
+            IsToggledStateActive = IsToggled.FindPropertyRelative("active");
 
-            selectThreshold = SetUpProperty(nameof(selectThreshold));
-            deselectThreshold = SetUpProperty(nameof(deselectThreshold));
+            SelectThreshold = SetUpAutoProp(nameof(SelectThreshold));
+            DeselectThreshold = SetUpAutoProp(nameof(DeselectThreshold));
 
-            toggleMode = SetUpProperty(nameof(toggleMode));
-            triggerOnRelease = SetUpProperty(nameof(triggerOnRelease));
+            ToggleMode = SetUpAutoProp(nameof(ToggleMode));
+            TriggerOnRelease = SetUpAutoProp(nameof(TriggerOnRelease));
 
             allowSelectByVoice = SetUpProperty(nameof(allowSelectByVoice));
             speechRecognitionKeyword = SetUpProperty(nameof(speechRecognitionKeyword));
```

Comment:
Should these be converted as well?

Reply:
Those are the ones that can't be autoprops! Those are still manual backing fields, because they have custom setters.